Twitter/Connector/packages.config

```diff
@@ -0,0 +1,53 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="AngularJS.Cookies" version="1.7.5" targetFramework="net461" />
+  <package id="AngularJS.Core" version="1.7.5" targetFramework="net461" />
+  <package id="bootstrap" version="4.2.1" targetFramework="net461" />
+  <package id="jQuery" version="3.0.0" targetFramework="net461" />
+  <package id="Microsoft.ApplicationInsights" version="2.8.1" targetFramework="net461" />
+  <package id="Microsoft.ApplicationInsights.Agent.Intercept" version="2.4.0" targetFramework="net461" />
+  <package id="Microsoft.ApplicationInsights.DependencyCollector" version="2.8.1" targetFramework="net461" />
+  <package id="Microsoft.ApplicationInsights.PerfCounterCollector" version="2.8.1" targetFramework="net461" />
+  <package id="Microsoft.ApplicationInsights.TraceListener" version="2.8.1" targetFramework="net461" />
+  <package id="Microsoft.ApplicationInsights.Web" version="2.8.1" targetFramework="net461" />
+  <package id="Microsoft.ApplicationInsights.WindowsServer" version="2.8.1" targetFramework="net461" />
+  <package id="Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel" version="2.8.1" targetFramework="net461" />
+  <package id="Microsoft.AspNet.Mvc" version="5.2.3" targetFramework="net461" />
+  <package id="Microsoft.AspNet.Razor" version="3.2.3" targetFramework="net461" />
+  <package id="Microsoft.AspNet.TelemetryCorrelation" version="1.0.4" targetFramework="net461" />
+  <package id="Microsoft.AspNet.Web.Optimization" version="1.1.3" targetFramework="net461" />
+  <package id="Microsoft.AspNet.WebApi.Client" version="5.2.7" targetFramework="net461" />
+  <package id="Microsoft.AspNet.WebApi.Core" version="5.2.7" targetFramework="net461" />
+  <package id="Microsoft.AspNet.WebApi.Owin" version="5.2.7" targetFramework="net461" />
+  <package id="Microsoft.AspNet.WebApi.WebHost" version="5.2.6" targetFramework="net461" />
+  <package id="Microsoft.AspNet.WebPages" version="3.2.3" targetFramework="net461" />
+  <package id="Microsoft.Azure.KeyVault.Core" version="1.0.0" targetFramework="net461" />
+  <package id="Microsoft.CodeDom.Providers.DotNetCompilerPlatform" version="2.0.1" targetFramework="net461" />
+  <package id="Microsoft.Data.Edm" version="5.7.0" targetFramework="net461" />
+  <package id="Microsoft.Data.OData" version="5.7.0" targetFramework="net461" />
```

Comment:
This version will crib. Not a blocking issue
Upgrade it in next PR. Similarly in other projects as well

Reply:
Will update in next PR